sdl2-vga-terminal/test/tests.cpp

```diff
@@ -56,40 +38,57 @@ TEST(VgaTerminal, CannotInit)
 	Environment::setUp();
 }
 
+TEST(VgaTerminal, TimerNotInitedWarning)
+{
+	Environment::tearDown();
+	ASSERT_EQ(0, SDL_Init(SDL_INIT_VIDEO));
+	SDL_LogSetPriority(SDL_LOG_CATEGORY_SYSTEM, SDL_LogPriority::SDL_LOG_PRIORITY_VERBOSE);
+	{
+		testing::internal::CaptureStderr();
+		std::string title = ::testing::UnitTest::GetInstance()->current_test_info()->name();
+		VgaTerminal term = VgaTerminal(title, SDL_WINDOW_HIDDEN, -1, 0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'term' is assigned a value that is never used.](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5427759)